client/components/LearnerViewDirective.js

```diff
@@ -370,6 +379,13 @@ tie.directive('learnerView', [function() {
           ].join('\n')
         ];
 
+        var SECONDS_TO_MILLISECONDS = 1000;
+        // Default time interval in seconds that codes will be auto saved.
+        // Codes will be auto-saved every minute.
+        var DEFAULT_AUTO_SAVE_SECONDS = 60;
```

Comment:
Consider AUTO_SAVE --> AUTOSAVE here and elsewhere. It appears to be one word: https://en.wikipedia.org/wiki/Autosave